Fixes gcc-3.4.0 compilation errors

rpc_hout.c: In function `print_funcdef':
rpc_hout.c:140: error: label at end of compound statement
make[2]: *** [/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/build-glibc/sunrpc/rpc_hout.o] Error 1
make[2]: Leaving directory `/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/glibc-2.1.3/sunrpc'

rpc_cout.c: In function `emit_inline':
rpc_cout.c:743: error: label at end of compound statement
...

--- glibc-2.1.3/sunrpc/rpc_hout.c.old	Sun Apr 11 22:06:26 2004
+++ glibc-2.1.3/sunrpc/rpc_hout.c	Sun Apr 11 22:06:42 2004
@@ -136,7 +136,7 @@
       pprogramdef (def);
       break;
     default:
-      /* ?... shouldn't happen I guess */
+      ; /* ?... shouldn't happen I guess */
     }
 }
 
--- glibc-2.1.3/sunrpc/rpc_cout.c.old	Sun Apr 11 22:10:39 2004
+++ glibc-2.1.3/sunrpc/rpc_cout.c	Sun Apr 11 22:10:53 2004
@@ -739,7 +739,7 @@
       f_print (fout, "}\n");
       break;
     default:
-      /* ?... do nothing I guess */
+      ; /* ?... do nothing I guess */
     }
 }
 
